README.md

```diff
@@ -30,7 +30,8 @@ FireCloud Orchestration Service
 * New paths to external endpoints should be added to `src/main/resources/configurations.conf`. Existing endpoint URLs are configured in `application.conf` and `test.conf`
 
 ## Building and Running
-See https://github.com/broadinstitute/firecloud-environment/blob/master/README.md for directions on running locally in docker.
+
+See https://github.com/broadinstitute/firecloud-develop for directions on running locally within Broad's DSDE environment.
```

Comment:
Is this okay?
https://github.com/broadinstitute/firecloud-develop/commit/e6df0729576eb0cefa2fd74698d2287936a1708e